.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "singleQuote": true,
```

Comment:
Can you update it to use `double quotes`?

Reply:
Alright, this is resolved in the newest commit.